lib/server/service.tsx

```diff
@@ -265,17 +272,34 @@ export async function acceptInvite(userId: string, inviteId: string) {
   return profile;
 }
 
+async function getUserCountSubquery() {
+  const userCount = db
+    .select({
+      tenantId: schema.profiles.tenantId,
+      userCount: sql<number>`COALESCE(COUNT(*) FILTER (WHERE ${schema.profiles.role} != 'guest'), 0)`
+        .mapWith(Number)
+        .as("user_count"),
+    })
+    .from(schema.profiles)
+    .groupBy(schema.profiles.tenantId)
+    .as("user_counts");
```

Comment:
You probably want to scope by the userId.  I think this retrieves user_counts for all tenants.

Reply:
Also wondering why we can't just do a simple count.  We should be able to use there where clause to filter the role.  `COALESCE(COUNT(*) FILTER (WHERE ${schema.profiles.role} != 'guest'), 0)` seems too complicated.